doc/modules/preprocessing.rst

```diff
@@ -240,9 +222,9 @@ data.
 
 .. topic:: Scaling a 1D array
 
-   All above functions (i.e. :func:`scale`, :func:`minmax_scale`,
-   :func:`maxabs_scale`, and :func:`robust_scale`) accept 1D array which can be
-   useful in some specific case.
+   The module provides convenience functions (i.e. :func:`scale`,
+   :func:`minmax_scale`, :func:`maxabs_scale`, and :func:`robust_scale`),
+   which accept 1D array and can be useful in some specific case.
```

Comment:
If we decide to keep this topic (I am -1), we have to add a warning note:

```suggestion
   The module provides convenience functions (i.e. :func:`scale`,
   :func:`minmax_scale`, :func:`maxabs_scale`, and :func:`robust_scale`),
   which accept 1D array and can be useful in some specific case.
   
   .. warning:: Risk of data leak
        Do not use these functions unless you know what you are
        doing. A common mistake is to apply it to the entire data
        *before* splitting into training and test sets. This will bias
        the model evaluation because information would have leaked
        from the test set to the training set. In general, we recommend
        using the transformers within a :class:`~sklearn.pipeline.Pipeline`
        in order to prevent most risks of data leaking.
```

Reply:
I'll just remove altogether